primitive.cabal

```diff
@@ -54,7 +54,7 @@ Library
         Data.Primitive.Internal.Operations
 
   Build-Depends: base >= 4.5 && < 4.15
-               , deepseq
+               , deepseq >= 1.4 && < 1.5
```

Comment:
Why do we need such tight bounds?

Reply:
I think we could technically allow `>= 1.1 && < 1.5`, but I'm not sure what the point would be.

Reply:
After the CI failure, I now see what the point would be.

Reply:
Yeah, don't mess with build plans people may need unless you have a good reason.